packages/ember-engines/README.md

```diff
@@ -19,6 +19,9 @@ perspective.
 ember install ember-engines
 ```
 
+## Upgrade guide
+
+* [0.10.0 -> 0.11.0](UPGRADE.md#0110)
```

Comment:
@void-mAlex there is `Migrating` section in https://ember-engines.com/docs/ so I believe we should keep documentation consolidated

Reply:
happy to consolidate there as well, I don't see how this extra doc hurts